linodecli/configuration.py

```diff
@@ -147,9 +147,10 @@ def configure(self, username=None):
             self.config.add_section(username)
 
         for k, v in config.items():
-            self.config.set(username, k, v)
-            if is_default:
-                self.config.set('DEFAULT', k, v)
+            if v:
```

Comment:
alternately: `if len(v)` if we are worried about future boolean options being possibly misinterpreted